src/v/kafka/server/tests/fetch_bench.cc

```diff
@@ -61,43 +62,48 @@ static constexpr auto small_fetch_config = fetch_bench_config{
   .batch_overhead = 70,
 };
 
+struct fetch_part {
```

Comment:
Are these all rf=1 topics?

Reply:
Yeah, all are rf=1. It's a single node RP cluster as well.